### ISSUE TITLE
Feature: graphical dynamics painter

### ISSUE THREAD

**Issue by brenthuisman** [arbor-sim/gui#17]
In addition to writing S-expressions, graphical delineation (use a mouse to 'draw' a region and generate an S-expression consequently) of regions and assigning dynamics (and parameters, etc) would be of great help designing cells.

**Comment by thorstenhater** [comment]
You mean like the literal region rendering we are doing since day 0?

**Comment by brenthuisman** [comment]
No, we meant using a mouse to 'draw' a region and generate an S-expression consequently.

**Comment by thorstenhater** [comment]
I think that's an insane amount of work for questionable payoff, but go for it, if you have the bandwidth.

**Comment by brenthuisman** [comment]
According to @llandsmeer this would be very useful, and he'd be the person to code it up (see assignees). Whether or not it's a prio or a nice-to-have-in-distant-future is something we can decide later.
